library/common/src/main/java/com/epmedu/animeal/common/di/AppModule.kt

```diff
@@ -32,4 +34,8 @@ internal object AppModule {
             produceFile = { context.preferencesDataStoreFile(DATA_STORE_PREFERENCES_KEY) }
         )
     }
+
+    @Singleton
```

Comment:
I don't think that this repository should be a singleton. I suggest moving it to home module and make it viewModelScoped